public/css/menFashion.css

```diff
@@ -0,0 +1,141 @@
+body{
+margin: 0px;
+padding: 0px;
+height: 100vh;
+width: 100%;
+}
+
+.container{
+  width: 1140px;
+  flex-wrap: wrap;
+
+}
+.containernav{
```

Comment:
you should choose method naming classes camel case or other and use it in all naming  plz